src/clipboard.js

```diff
@@ -1,11 +1,28 @@
-import ClipboardAction from './clipboard-action';
+/* eslint-disable class-methods-use-this */
```

Comment:
I guess would make more sense to add this to the eslint ignore rules and create a comment on why this needs to be there, what you say?

Reply:
Sure, I will do.